### PR TITLE
Added Bucket Sort in GO language

### LINKED ISSUE #2124: Bucket Sort
- [x] C
- [x] C++
- [ ] C# - @luv4bytes
- [ ] CoffeeScript
- [ ] Dart
- [x] Go - @m-code12
- [x] Java
- [x] JS - @abhu-A-J
- [x] Kotlin - @shubhammantri1
- [ ] Php - @srimani-programmer
- [x] Python
- [x] Readme
- [x] Ruby - @m-code12
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by AkanshaKamboj**
Fixes #2124 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by AkanshaKamboj** [comment]
@jainaman224 I have completed the code for Bucket Sort in GO language. Please merge my PR or suggest some changes if needed.

File: Bucket_Sort/Bucket_Sort.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import (
+	"fmt"
+	"sort"
+)
+
+// Function to sort the array elements using Bucket sort
+func bucket_sort(arr []float64, size int)
+{
+	var max_ele, min_ele float64
+    min_ele = arr[0]
```
Comment (MastersAbh) [inline Bucket_Sort/Bucket_Sort.go:12]: please fix indentation

File: Bucket_Sort/Bucket_Sort.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
+package main
```
Comment (MastersAbh) [inline Bucket_Sort/Bucket_Sort.go:1]: add problem description at start of file

File: Bucket_Sort/Bucket_Sort.go [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+// Function to sort the array elements using Bucket sort
+func bucket_sort(arr []float64, size int)
+{
+	var max_ele, min_ele float64
+    min_ele = arr[0]
+
+    //Find the minimum and maximum element in the array
+	for i:= 0; i < size; i++ { 
+
+		if arr[i] < min_ele {
```
Comment (MastersAbh) [inline Bucket_Sort/Bucket_Sort.go:17]: TAB has been used for indentation in this file, Can you use 4 space indentation ?

File: Bucket_Sort/Bucket_Sort.go [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+	Enter the size of the array:10                                                                                                         
+	3 9 11 7 2 8 13 5 12 4
+	Array before sorting: [3 9 11 7 2 8 13 5 12 4]
+
+Sample Output
+
+	Array after sorting: [2 3 4 5 7 8 9 11 12 13] 
+*/
+
+
```
Comment (MastersAbh) [inline Bucket_Sort/Bucket_Sort.go:92]: Please remove the extra blank lines

**Comment by MastersAbh** [comment]
Hi @AkanshaKamboj are you working on this ?

**Comment by AkanshaKamboj** [comment]
Hi @MastersAbh
Was busy with some internship work. Will complete it by today itself and inform you. 

Thank you

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**